benchmarks/asv.conf.json

```diff
@@ -28,7 +28,7 @@
     // If missing or the empty string, the tool will be automatically
     // determined by looking for tools on the PATH environment
     // variable.
-    "environment_type": "virtualenv",
+    "environment_type": "conda",
```

Comment:
this seems unnecessary and unrelated to the rest of this PR, can you undo the change?